packages/analytics/dashboard-renderer/src/components/DashboardRenderer.vue

```diff
@@ -215,6 +216,11 @@ const handleUpdateTiles = (tiles: GridTile<TileDefinition>[]) => {
       definition: tile.meta,
     } as TileConfig
   })
+  const newGridSize = {
+    cols: Math.max(...updatedTiles.map(t => t.layout.position.col + t.layout.size.cols)),
+    rows: Math.max(...updatedTiles.map(t => t.layout.position.row + t.layout.size.rows)),
+  }
+  model.value.gridSize = newGridSize
```

Comment:
config is now bound using v-model, enabling two-way data binding. Changes made internally will automatically sync with the parent component.